readme.txt

```diff
@@ -334,6 +334,7 @@ Please see the changelog for the complete list of changes in this release. Remem
 * Tweak - Tweaked some language in Event Aggregator's metabox on individual edit-event screens to reduce confusion around the impact of the Update Authority on CSV imports [77957]
 * Tweak - Added new filter: `tribe_events_force_filtered_ical_link`. This makes the "Export Events" URL more easily modifiable (thanks to @tdudley07 for highlighting this issue) [43908]
 * Tweak - Made the "End of Day Cutoff" option better accommodate 24-hour and other time formats (thanks @festivalgeneral for bringing this issue to our attention!) [78621]
+* Tweak - Made the options presented by the timezone selector filterable (our thanks to National University's Marketing Department for this idea) [92909]
```

Comment:
can you call out the new filter by name here? `tribe_events_timezone_choice`

Reply:
good call: adding